packages/client/lib/commander.ts

```diff
@@ -15,6 +15,11 @@ interface AttachConfigOptions<
   config?: CommanderConfig<M, F, S, RESP>;
 }
 
+/* FIXME: better error message / link */
```

Comment:
isn't this ready?

Reply:
the error message is better now, unsure its as good as could be?  If its fine, can remove the comment